recipes/libavrocpp/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+
+class LibavrocppConan(ConanFile):
+    name = "libavrocpp"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Avro is a data serialization system."
+    homepage = "https://avro.apache.org/"
+    topics = ("serialization", "deserialization")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_snappy": [True, False],
+    }
+    default_options = {"shared": False, "fPIC": True, "with_snappy": True}
+    generators = "cmake", "cmake_find_package"
+
+    @property
+    def _source_subfolder(self):
+        return os.path.join("source_subfolder", "lang", "c++")
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        tools.replace_in_file(
+            os.path.join(self._source_subfolder, "CMakeLists.txt"),
+            "project (Avro-cpp)",
+            """project (Avro-cpp)
+               include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+               conan_basic_setup()""",
+        )
+
+        if self.options.with_snappy:
+            tools.replace_in_file(
+                os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                "find_package(Snappy)",
+                "",
+            )
+        self._cmake = CMake(self)
+        self._cmake.configure(source_folder=self._source_subfolder)
+        return self._cmake
+
+    def configure(self):
```

Comment:
Oh, I thought I fixed that. Are you referring to the extra `_configure_cmake()` method? The other ones are in the correct order, otherwise I have overlooked something. I fixed that in a previous commit.

Reply:
Having problems with the UI this morning....

I forgot to submit this last night so it might be stale